src/main.cpp

```diff
@@ -573,8 +575,7 @@ void loop()
                 strOutput += str_sprintf("Buffer: %d/%d, ", bufferManager.Depth(), bufferManager.BufferCount());
             #endif
 
-            auto& taskManager = g_ptrSystem->TaskManager();
-            strOutput += str_sprintf("CPU: %03.0f%%, %03.0f%%, FreeDraw: %4.3lf", taskManager.GetCPUUsagePercent(0), taskManager.GetCPUUsagePercent(1), g_Values.FreeDrawTime);
+            strOutput += str_sprintf("CPU: %03.0f%%, %03.0f%%, FreeDraw: %4.3lf", g_ptrSystem->TaskManager().GetCPUUsagePercent(0), g_ptrSystem->TaskManager().GetCPUUsagePercent(1), g_Values.FreeDrawTime);

```

Comment:
This change genuinely interests me. Why did you choose to replace the one reference variable with two invocations of the getter?

Reply:
There was somethig lint on my laptop didn't like about it.  But it passes cppcheck and looks fine to me as it was, so I've put it back!

Reply:
Ah, now I remember.  I can make it a const auto reference, since all the member calls are const.  So now I have.